guides/pricing_and_taxation.md

```diff
@@ -0,0 +1,127 @@
+
+### Tax Rates
+
+A tax rate is essentially a percentage amount charged based on the net price. Tax rates also contain other important information.
+
+* Whether product prices are inclusive of this tax
+* The zone within which the order address must fall
+* The tax category that a product must belong to in order to be considered taxable.
+
+Solidus will calculate tax based on all matching tax rates for the order's tax address. It's also possible to have more than one applicable tax rate for a single zone. In order for a tax rate to apply to a particular product, that product must have a tax category that matches the tax category of the tax rate.
+
+***
+To best appreciate how Solidus' taxation system works, have a look at the [taxation integration specs](https://github.com/solidusio/solidus/blob/master/core/spec/models/spree/tax/taxation_integration_spec.rb).
+***
+### Sales tax vs. Value-Added Tax
+
+In e-Commerce, the type of tax we care most about is consumption tax, of which two main models are found: Sales Tax and Value Added Tax (VAT).
+
+The Sales tax policies found in the USA are *additional* taxes. They do not change the amount of a line item, but are added just before calculating the order total. For Sales tax, the prices for items do not change when adding them to an order.
+
+With Value-Added Tax (VAT), as found in Europe, but also in many other parts of the world, the prices of products and line items need to be shown *including* the consumption tax. When a product is exported, the price of that product has to change: It will be lower, because there is no included tax anymore.
+
+***
+Most taxes can be considered one of these two types. For instance, in Australia customers pay a Goods and Services Tax (GST). This is basically equivalent to VAT in Europe.
+***
+
+To model these behaviours in Solidus, use the `included_in_price` boolean on the `Spree::TaxRate` model. If `included_in_price` is set to `true`, Solidus will assume VAT-style taxation for this tax rate, if it is set to `false`, it will assume Sales tax style taxation.
+
+In some cases you may need to charge one type of tax for orders falling within one zone and another type of tax for orders falling within a different zone. There are even some rare situations where you may need to charge both types of tax in the same zone. Solidus  supports all of these scenarios.
+
+***
+Taxation within the United States can get exceptionally complex, with different states, counties and even cities having different taxation rates. If you are shipping interstate within the United States, we would strongly advise you to use the [Solidus Avatax](https://github.com/solidusio/solidus_avatax) extension so that you get correct tax rates.
```

Comment:
I'd also like to see here a link to the open issue @jordan-brough is working on in https://github.com/solidusio/solidus/issues/1252 to suggest that there is more than one solution to this, and it is under active development.